docs/import/add_disaster.js

```diff
@@ -0,0 +1,220 @@
+import {eeLegacyPathPrefix, gdEePathPrefix} from '../ee_paths.js';
+import {getFirestoreRoot} from '../firestore_document.js';
+
+export {
+  enableWhenReady,
+};
+// Visible for testing
+export {
+  addDisaster,
+  createOptionFrom,
+  createStateAssetPickers,
+  disasters,
+  emptyCallback,
+  writeDisaster,
+};
+
+// Currently a map of disaster name to states. This pulls once on firebase
+// authentication and then makes local updates afterwards so we don't need to
+// wait on firebase writes to read new info.
+const disasters = new Map();
+
+const SENTINEL_OPTION_VALUE = '...';
+const SENTINEL_NEW_DISASTER_VALUE = 'NEW DISASTER';
+
+/**
+ * Populates the disaster picker with disasters from firestore, and enables
```

Comment:
I think you can set the size of the disaster picker so that it doesn't jump around. We don't know how long disaster names will be, but can make a pretty good guess?

Reply:
Set at 160 px for now - not very responsive but I'm ok with that as a first go

Reply:
I'm still seeing it very small, at least in my browser that is throwing an authentication error. You can see it small if you open this page in an incognito window.

![image](https://user-images.githubusercontent.com/10134896/68816784-7bf31c80-064d-11ea-8efa-4ac4f0ec6e3a.png)

Reply:
I actually don't see it as small from the GD account, or from an incognito page, or my corp page. It's always 160 px.